vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb

```diff
@@ -20,9 +20,18 @@ def initialize
     end
 
     # GET /facilities/:facility_id/occupancies
-    # Provided by NewInprocessController
-    # def index
-    # end
+    def index
+      order_details = new_or_in_process_orders.joins(:order)
+
+      @search_form = TransactionSearch::SearchForm.new(params[:search], defaults: { date_range_field: "ordered_at" })
+      @search = TransactionSearch::Searcher.new(TransactionSearch::ProductSearcher).search(order_details, @search_form)
```

Comment:
I'll be honest, I'm a little confused by them wanting to remove the ordered for filter. I'm also a little surprised they wouldn't want the payment source filter on both in/inprocess and problems.

Reply:
I thought that was weird too, but I wasn't sure it was worth pushing back on.

Reply:
Yeah, probably not worth fighting. And it's easy enough to add back in when they realize they want it.